Sources/Graphiti/Component/Component.swift

```diff
@@ -1,18 +1,23 @@
 import GraphQL
 
-open class Component<Resolver, Context> {
+open class Component<Resolver, Context>: ExpressibleByStringLiteral {
     let name: String
     var description: String? = nil
     
     init(name: String) {
         self.name = name
     }
     
+    public required init(stringLiteral string: StringLiteralType) {
+        self.name = ""
+        self.description = string
```

Comment:
Are those the description that end up becoming GraphQL documentation? Could you add an example in the docs on how to use this one?

Also, why is the `name` property empty here? Can't we initialize a field with the required name and a description?

Reply:
I found out about the `description` modifier. Never mind this one.